tests/Integration/ImmediateQRCode/IssueImmediateQRCodeTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Integration\ImmediateQRCode;
+
+use Tests\TestCase;
+use Tests\Trait\HttpClientMock;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\Loc;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\Value;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\Debtor;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\Calendar;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\InputData;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\Information;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\IssueImmediateQRCode;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\ItauIssueImmediateQRCodeException;
+use AstrotechLabs\Itau\ImmediateQRCode\IssueImmediateQRCode\Dto\AdditionalInformationCollection;
+
+class IssueImmediateQRCodeTest extends TestCase
+{
+    use HttpClientMock;
+
+    public function testMustReturnTheInformationAfterIssuingTheQrCodeWithTheMinimumNumberOfFieldsFilledIn()
```

Comment:
Qual é a informação em 'the information'? Acho que da forma que está escrito fica pedindo esclarecimento. Sugiro mudar para algo como testReceivesQrCodeInfoWhenIssuingQrCodeWithMinimumFields